splunk_logger.go

```diff
@@ -52,13 +52,19 @@ const (
 	defaultBufferMaximum = 10 * defaultPostMessagesBatchSize
 	// Number of messages allowed to be queued in the channel
 	defaultStreamChannelSize = 4 * defaultPostMessagesBatchSize
+	// Partial log hold duration (if we are not reaching max buffer size)
+	defaultTempMsgBufferHoldDuration = 100 * time.Millisecond
```

Comment:
is this duration a little bit too short ?

Reply:
See above.